spec/features/enable_buttons_spec.rb

```diff
@@ -1,25 +1,12 @@
 require 'spec_helper'
 
+# Buttons are enabled in app/helpers/argo_helper#render_buttons.
+
 feature 'Enable buttons' do
-  before do
-    @current_user = double(
-      :webauth_user,
-      login: 'sunetid',
-      logged_in?: true,
-      permitted_apos: [],
-      is_admin: true,
-      roles: []
-    )
-    @obj = double(
-      'item',
-      admin_policy_object: false,
-      datastreams: {},
-      can_manage_item?: true
-    )
-    allow_any_instance_of(ApplicationController).to receive(:current_user).
-      and_return(@current_user)
-    allow(Dor).to receive(:find).and_return(@obj)
+  before :each do
+    admin_user # see spec_helper
   end
+
```

Comment:
Use a vanilla admin user and a regular fixture (a Dor fixture will throw a spanner at `User#get_dor_object`).